cli/dcoscli/package/main.py

```diff
@@ -390,24 +388,16 @@ def _list(json_, endpoints, app_id, package_name):
     :rtype: int
     """
 
-    package_manager = _get_package_manager()
-    installed = package.installed_packages(package_manager, endpoints)
+    if endpoints:
+        emitter.publish(
+            errors.DefaultError("The --endpoints flag is deprecated"))
```

Comment:
I see. We continue but log this to stderr, right? Let's file an GitHub issue to remove this for v0.6.0